modules/ocf_ssh/manifests/environment.pp

```diff
@@ -0,0 +1,6 @@
+class ocf_ssh::environment {
+  file { '/etc/motd':
+    ensure => link,
+    target => '/home/s/st/staff/motd',
+  }
+}
```

Comment:
suggestion: move to `ocf::motd` and `include ocf::motd` inside `ocf_ssh` and `ocf_apphost`